build/tutorials-contest/liquid-avax/Backend/backend.js

```diff
@@ -0,0 +1,272 @@
+// This script will be used to do the cross-chain transfer of the owner wallet from C to P chain and stake them
+// to the user-chosen node.
+
+import fs from "fs";
+import {Avalanche, BinTools, BN} from "avalanche";
+import Web3 from "web3";
+import { Buffer } from "buffer";
+
+let xChain, xKeyChain, xChainAddress;
+let cChain, cKeyChain, cChainAddress;
+let pChain, pKeyChain, pChainAddress;
+
+let xChainBlockchainID, cChainBlockchainID, pChainBlockchainID;
+
+let web3;
+
+let contractAbi;
+
+let masterAddress = "0x7bD7A7D2Ba70db40740780828b236F9246BB7F78";
+let privateKey = JSON.parse(await fs.readFileSync("./data.json")).privateKey;
+
+let currentTxId = [];
+
+let utxoset;
+
+
+async function waitForStaked() {
```

Comment:
Use the fat arrow syntax instead of the `function` keyword.

```ts
async function waitForStaked() {}
```

becomes

```ts
const waitForStaked = async (): Promise<any> => {}
```

Note it's a `const` w/ fat arrow and return type `Promise<any>`